### PR TITLE
[OAP-TOOLS-1]Add docs and scripts

### PR DESCRIPTION
**Pull request by HongW2019**
Add docs and scripts in order to build documentation on website.

### CODE REVIEW

**Comment by HongW2019** [comment]
@yao531441  Please review and help merge it.

File: docs/docs/OAP-Developer-Guide.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- [OneAPI](https://software.intel.com/content/www/us/en/develop/tools/oneapi.html)
+- [Arrow](https://github.com/Intel-bigdata/arrow)
+
+- **Requirements for Shuffle Remote PMem Extension**  
+If enable Shuffle Remote PMem extension with RDMA, you can refer to [PMem Shuffle](https://github.com/oap-project/pmem-shuffle) to configure and validate RDMA in advance.
+
+We provide scripts below to help automatically install dependencies above **except RDMA**, need change to **root** account, run:
+
+```
+# git clone -b <tag-version>
```
Comment (yao531441) [inline docs/docs/OAP-Developer-Guide.md:36]: https://github.com/oap-project/oap-tools

File: docs/mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+site_name: OAP Project
+
+repo_name: 'Fork on GitHub '
+repo_url: https://github.com/HongW2019/hongw2019.github.io.git
```
Comment (yao531441) [inline docs/mkdocs.yml:4]: Are you sure to use a personal repo?

File: docs/docs/OAP-Developer-Guide.md (hunk 1 above)
Comment (HongW2019) [reply docs/docs/OAP-Developer-Guide.md]: done, thanks

**Pull request merged**